opencog/persist/zmq/atomspace/ZMQClient.h

```diff
@@ -1,11 +1,11 @@
 /*
- * opencog/atomspace/ZMQClient.h
+ * FUNCTION:
+ * Base class for ZeroMQ-backed persistent storage.
  *
- * Copyright (C) 2008-2010 OpenCog Foundation
- * All Rights Reserved
- *
- * Written by Erwin Joosten
+ * HISTORY:
+ * Copyright (c) 2015 Erwin Joosten, Hendy Irawan <ceefour666@gmail.com>
```

Comment:
I am pretty sure Erwin did not work on this in 2015!  He probbly worked on this in 2009 or 2010 and then assigned copyright to the OpenCog Foundation .. whose copyright you removed ...